kalite/contentload/management/commands/channels/base.py

```diff
@@ -53,11 +50,12 @@ def whitewash_node_data(node, path="", ancestor_ids=[], channel_data={}):
     node["path"] = node.get("path", "") or path + node["slug"] + "/"
     if "title" not in node:
         node["title"] = (node.get(channel_data["title_key"][kind], ""))
-    node["title"] = node["title"].strip()
+    node["title"] = (node["title"] or "").strip()
 
-    # Add some attribute that should have been on there to start with.
-    node["parent_id"] = ancestor_ids[-1] if ancestor_ids else None
-    node["ancestor_ids"] = ancestor_ids
+    if ancestor_ids:
```

Comment:
Failing to check for the existence of this causes all ancestor_id information to be wiped.